Dispatcher/Controllers/Homepage/HomeViewController.swift

```diff
@@ -23,11 +26,42 @@ class HomeViewController: UIViewController {
         articlesTableView.dataSource = self
         articlesTableView.delegate = self
         
-        articlesTableView.register(UINib(nibName: "ArticleCell", bundle: nil), forCellReuseIdentifier: ArticleCell.articleCellIdentifier)
+        articlesTableView.register(UINib(nibName: "ArticleCell", bundle: nil), forCellReuseIdentifier: AppConstants.articleCellIdentifier)
```

Comment:
We'd want to save the "ArticleCell" string as a constant and not write it manually when we need it